metricbeat/beater/metricbeat.go

```diff
@@ -12,6 +12,9 @@ import (
 
 	"github.com/elastic/beats/libbeat/cfgfile"
 	"github.com/pkg/errors"
+
+	//Add metricbeat specific processors
+	_ "github.com/elastic/beats/metricbeat/processors/annotate/kubernetes"
```

Comment:
We should auto generate this in the future for all beats based on the convention for the namespace and make it part of the `include` package.